### PR TITLE
Add Ambit3 Vertical support

### PR DESCRIPTION
**Pull request by krtek4**
This adds information about the Ambit3 Vertical so that it is recognized.

I hadn't had the opportunity to sync any run yet, but at least the watch is not displayed as supported and the sync of the rest of the information seems to be working.

Let me know if I need to do anything else for the support to be complete.

### CODE REVIEW

File: src/libambit/libambit.rules [hunk 1]
```diff
@@ -19,6 +19,9 @@ SUBSYSTEMS=="usb", ATTRS{idProduct}=="001a", ENV{openambit}="yes"
 # Emu (a.k.a Suunto Ambit3 Peak)
 SUBSYSTEMS=="usb", ATTRS{idProduct}=="001b", ENV{openambit}="yes"
 
+# Emu (a.k.a Suunto Ambit3 Vertical)
```
Comment (centic9) [inline src/libambit/libambit.rules]: The "Emu" here seems to be a small typo in the comment...

Comment (krtek4) [inline src/libambit/libambit.rules]: Yes, I forgot to change it in the rule once I determined what it was. It is now corrected.

**Comment by krtek4** [comment]
The moves does not seem to be synced from the watch :(

How can I check what is happening ?

**Comment by krtek4** [comment]
Here is some output from OpenAmbit in debug mode :

```
libambit WARNING: libambit_navigation_read(): Driver does not support navigation_waypoint_read
Start reading log... 
libambit WARNING: libambit_pmem20_log_init(): Failed to read first data chunk
Finished reading logs... I think...
Finished freeing data 1
Finished freeing data 2
End reading log... 
Start sport mode 
```
I tried looking at the code, but C isn't my forte and not really sure what it really means.

Also, I tried syncing with the desktop apps, my two runs weren't uploaded to Movescount, but a run I did this morning was, so either I am experiencing a but with Moveslink or somehow OpenAmbit marked the moves as already downloaded and Moveslink ignored them.

I'll try to have some dump this weekend.

**Comment by krtek4** [comment]
I recorded two moves on the watch this afternoon and uploaded them using Moveslink.

I tried dissecting the `.pcap` file, sadly the code of the dissector seems to be working only on Wireshark version 1.X which is not readily available on my distro. I tried using a Wireshark 1.12 version but I got a segmentation fault opening the file.

Anyhow, I am not really accustomed to wireshark, so here are my files, hoping someone with more expertise might be able to do something : http://files.crettenand.info/ambit3capture.zip

**Comment by wbarnard** [comment]
Support for Ambit 3 Vertical (plus Suunto Traverse and Ambit 3 fw v2.2.16) is now available in my branch https://github.com/wbarnard/openambit/tree/extramodels.

This is ready for testing, if no problems I will create a pull request.

**Comment by krtek4** [comment]
Hi @wbarnard,

The branch you linked to has a last commit in August, did you push your code ?

I'll gladly test, but I doesn't see anything related to the new models. What am I missing ?

**Comment by wbarnard** [comment]
Code only just pushed you should see the latest commit from today that has all the changes

**Comment by krtek4** [comment]
Great, I'll make a test this evening !

**Comment by krtek4** [comment]
Openambit is now able to read the logs on the watch. even those that the official app wasn't able to "see" due to previous tries.

I am currently having issues authenticating to movescount, but that is probably unrelated. I'll keep you posted !

Thanks @wbarnard, working really great sofar.

**Comment by krtek4** [comment]
I may have talked a bit too quickly. 

I fixed my authentication issue by removing my config file and authenticating again. but now I have the following errors:

```
asyncGet:  "https://uiservices.movescount.com/userdevices/35E919510F001000?appkey=HpF9f1qV5qrDJ1hY1QK1diThyPsX10Mh4JvCw9xVQSglJNLdcwr3540zFyLzIC3e&userkey=***********&email=************" 
asyncPost:  "https://uiservices.movescount.com/moves/?appkey=HpF9f1qV5qrDJ1hY1QK1diThyPsX10Mh4JvCw9xVQSglJNLdcwr3540zFyLzIC3e&userkey=************&email=************" 
Failed to upload log (err code: 202 ), movescount.com replied with " ""Device not found in the queue, so Move cannot be queued"" "

**Comment by krtek4** [comment]
Digging a bit the issue might be the serial number, OpenAmbit report it as being "35E919510F001000", as I can read in the interface and also in the `asyncGet` above.

But, this number cannot be seen anywhere on my watch, and I registred it using the serial "1622104930".

Is this related somehow ?

**Comment by Gerold-R** [comment]
@krtek4 This is ok but I don't know how these 2 numbers are related. 

BTW: The SN on the watch contains the year (16 = 2016) and week-number (22)

**Comment by centic9** [comment]
This can be closed. It was added via PR #180 already.

**Comment by mbernasocchi** [comment]
closing as added via #180

**Pull request closed without merging**